### DISCUSSION TITLE
Add Apple Pencil setting on iPad

### DISCUSSION THREAD

**Discussion by 27tj** [ppy/osu#24991] [Ideas]
While playing osu on iPad, I tried to use Apple Pencil to track and use my other hand to tap, but Apple Pencil screen tap will register a click and mess up my play. I wish there is setting for disable Apple Pencil click just like the mouse button did.

**Comment by steve-1507** [comment]
Currently it is possible to position the cursor using the Pencil while in play and click by tapping the screen, however if you tap anywhere near the circles, your finger tap will take over. There should be a feature to completely ignore finger taps for positioning and process it like clicking M1 (mouse button 1).
